sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneEventHubClient.cs

```diff
@@ -206,101 +205,103 @@ public override async Task<EventHubProperties> GetPropertiesAsync(CancellationTo
                 runtimeInformation.LastEnqueuedSequenceNumber,
                 runtimeInformation.LastEnqueuedOffset,
                 runtimeInformation.LastEnqueuedTimeUtc,
-                runtimeInformation.IsEmpty,
-                DateTime.UtcNow
+                runtimeInformation.IsEmpty
             );
         }
 
         /// <summary>
-        ///   Creates an event sender responsible for transmitting <see cref="EventData" /> to the
-        ///   Event Hub, grouped together in batches.  Depending on the <paramref name="senderOptions"/>
-        ///   specified, the sender may be created to allow event data to be automatically routed to an available
+        ///   Creates an Event Hub producer responsible for transmitting <see cref="EventData" /> to the
+        ///   Event Hub, grouped together in batches.  Depending on the <paramref name="producerOptions"/>
+        ///   specified, the producer may be created to allow event data to be automatically routed to an available
         ///   partition or specific to a partition.
         /// </summary>
         ///
-        /// <param name="senderOptions">The set of options to apply when creating the sender.</param>
+        /// <param name="producerOptions">The set of options to apply when creating the producer.</param>
         ///
-        /// <returns>An event sender configured in the requested manner.</returns>
+        /// <returns>An producer configured in the requested manner.</returns>
         ///
-        public override EventSender CreateSender(EventSenderOptions senderOptions)
+        public override EventHubProducer CreateProducer(EventHubProducerOptions producerOptions)
         {
             TrackOne.EventDataSender CreateSenderFactory()
```

Comment:
These should continue to be `Sender` nomenclature, as they refer to the track one types, which do not use the new nomenclature.